java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -312,4 +317,59 @@ CompletableFuture<Map<String, String[][]>> xrevrange(
      */
     CompletableFuture<Map<String, String[][]>> xrevrange(
             String key, StreamRange end, StreamRange start, long count);
+
+    /**
+     * Creates a new consumer group uniquely identified by <code>groupname</code> for the stream
+     * stored at <code>key</code>.
+     *
+     * @see <a href="https://valkey.io/commands/xgroup-create/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @param groupname The newly created consumer group name.
+     * @param id Stream ID that specifies the last delivered entry in the stream from the new group’s
+     *     perspective. The special ID <code>"$"</code> can be used to specify the last entry in the
+     *     stream.
+     * @return <code>OK</code>.
+     * @example
+     *     <pre>{@code
+     * // Create the consumer group "mygroup", using zero as the starting ID:
+     * assert client.xgroupCreate("mystream", "mygroup", "0-0").get().equals("OK");
+     * }</pre>
+     */
+    CompletableFuture<String> xgroupCreate(String key, String groupname, String id);
+
+    /**
+     * Creates a new consumer group uniquely identified by <code>groupname</code> for the stream
+     * stored at <code>key</code>.
+     *
+     * @see <a href="https://valkey.io/commands/xgroup-create/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @param groupname The newly created consumer group name.
+     * @param id Stream ID that specifies the last delivered entry in the stream from the new group’s
```

Comment:
```suggestion
     * @param id Stream entry ID that specifies the last delivered entry in the stream from the new group’s
```
Please apply thoughout (and spottless)

Reply:
ok